templates/artist/includes/artist_detail_countdown.html

```diff
@@ -12,8 +12,8 @@
                 <span class="goal-label">Funded</span>
             </div>
             <div class="goal-stat">
-                <span class="goal-number">${{ campaign.amount_raised|intcomma }}</span>
-                <span class="goal-label">Raised</span>
+                <span class="goal-number">${{ campaign.amount|intcomma }}</span>
```

Comment:
We should probably revert this change, since the `artist_detail_countdown` is now only used in the past campaigns section.